packages/amplication-git-pull-request-service/src/core/diff/diff.service.ts

```diff
@@ -43,52 +48,35 @@ export class DiffService {
       'Cant get the same build id'
     );
 
-    return this.firstBuild(newBuildPath);
-    // // return all the new files if an old build folder dont exist
-    // if (existsSync(oldBuildPath) === false) {
-    //   return this.firstBuild(newBuildPath);
-    // }
+    DiffService.assertBuildExist(oldBuildPath);
+    DiffService.assertBuildExist(newBuildPath);
 
-    // DiffService.assertBuildExist(newBuildPath);
+    this.logger.info({ oldBuildPath, newBuildPath });
 
-    // this.logger.info({ oldBuildPath, newBuildPath });
-
-    // const res = await compare(oldBuildPath, newBuildPath, {
-    //   compareContent: true,
-    //   compareDate: false,
-    //   compareSize: false,
-    //   compareSymlink: false,
-    // });
-
-    // this.logger.debug('Finish the dir-compare lib process');
+    const res = await compare(oldBuildPath, newBuildPath, {
+      compareContent: true,
+      compareDate: false,
+      compareSize: false,
+      compareSymlink: false,
+      skipEmptyDirs: true,
+    });
 
-    // const changedFiles = res.diffSet.filter((diff) => {
-    //   if (diff.state !== 'equal' && diff.type2 === 'file') {
-    //     //make sure that only new files enter and ignore old files
-    //     if (diff.state !== 'left') {
-    //       return true;
-    //     }
-    //   }
-    //   return false;
-    // });
+    this.logger.debug('Finish the dir-compare lib process');
 
-    // this.logger.info('The list of the changed files', { changedFiles });
+    if (!res?.diffSet) {
+      throw new Error('Error in creating a diff set');
+    }
 
-    // const modules = changedFiles.map(async (diff) => {
-    //   const path = join(diff.relativePath, diff.name2);
-    //   const code = await readFileSync(join(diff.path2, diff.name2)).toString(
-    //     'utf8'
-    //   );
-    //   return {
-    //     path,
-    //     code,
-    //   };
-    // });
+    const modules = mapDiffSetToPrModule(res.diffSet, [deleteFilesVisitor]);
 
-    // return await Promise.all(modules);
+    const resultModule = await Promise.all([
+      ...(await this.allNewModules(newBuildPath)),
+      ...modules,
+    ]);
+    return resultModule;
   }
 
-  private async firstBuild(newBuildPath: string) {
+  private async allNewModules(newBuildPath: string) {
```

Comment:
`allNewModules` refers to a noun, but this is a method.

What are we doing here? Reflect that in the name of the method please.

Also, in the context of this method, the `new` in `newBuildPath` is meaningless. It is the only build path the method is aware of. I suggest renaming it to `buildPath`.